History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
```

Comment:
Maybe explain that this is on the server side. And what exactly means "bound"? To `this`? Or something else. ;-) (I know what it is, I am just saying that it is unclear for readers.)

BTW, what is `this` bound to inside callbacks? Should we bound them to something?